commit 2346d0cde544179a8d235375f1bfbca5c141a31b
Author: Travis Place <wishie@wishie.net>
Date:   Mon Sep 1 08:24:00 2008 +0200

    ALSA: hda - Add model for Toshiba L305 laptop
    
    Added Subsystem IDs (0x1179, 0xff64) for the Toshiba Satellite L305
    laptop, so it automatically uses the ALC268_TOSHIBA quirk.
    
    Signed-off-by: Travis Place <wishie@wishie.net>
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Jaroslav Kysela <perex@perex.cz>

Adjusted to apply to Debian's 2.6.26 by dann frazier <dannf@debian.org>

diff -urpN linux-source-2.6.26.orig/sound/pci/hda/patch_realtek.c linux-source-2.6.26/sound/pci/hda/patch_realtek.c
--- linux-source-2.6.26.orig/sound/pci/hda/patch_realtek.c	2009-02-08 23:41:17.000000000 -0700
+++ linux-source-2.6.26/sound/pci/hda/patch_realtek.c	2009-02-09 00:09:22.000000000 -0700
@@ -10700,6 +10700,7 @@ static struct snd_pci_quirk alc268_cfg_t
 	SND_PCI_QUIRK(0x1043, 0x1205, "ASUS W7J", ALC268_3ST),
 	SND_PCI_QUIRK(0x1179, 0xff10, "TOSHIBA A205", ALC268_TOSHIBA),
 	SND_PCI_QUIRK(0x1179, 0xff50, "TOSHIBA A305", ALC268_TOSHIBA),
+	SND_PCI_QUIRK(0x1179, 0xff64, "TOSHIBA L305", ALC268_TOSHIBA),
 	SND_PCI_QUIRK(0x14c0, 0x0025, "COMPAL IFL90/JFL-92", ALC268_TOSHIBA),
 	SND_PCI_QUIRK(0x152d, 0x0763, "Diverse (CPR2000)", ALC268_ACER),
 	SND_PCI_QUIRK(0x152d, 0x0771, "Quanta IL1", ALC267_QUANTA_IL1),
